qt/python/mantidqt/dialogs/spectraselectordialog.py

```diff
@@ -66,12 +67,15 @@ class SpectraSelectionDialog(SpectraSelectionDialogUIBase):
     @staticmethod
     def get_compatible_workspaces(workspaces):
         matrix_workspaces = []
+        md_workspaces = list()
```

Comment:
This list looks like it might have been left behind from a previous change and is not longer required?